src/models/PenaltyReferenceDetails.ts

```diff
@@ -0,0 +1,11 @@
+export class PenaltyReferenceDetails  {
```

Comment:
Technically that could be just an interface cause currently class does not add any behaviour, it is just data bucket.